core/src/data_sources/data_source.rs

```diff
@@ -759,6 +980,15 @@ impl DataSource {
                 .unwrap_or(std::cmp::Ordering::Equal)
         });
 
+        // Order chunks by offset.
```

Comment:
Interesting this changes the current interface. They are oredered by score as guaranteed by qdrant. Let's preserve this interface if you do need to resort then let's resort by `score` and make sure it's descending